lib/autoBlock.js

```diff
@@ -0,0 +1,215 @@
+var _ = require('lodash')
+var async = require('async')
+var httpStatus = require('http-status')
+
+var utils = require('./utilities')
+
+// TODO reject invalid options/flags (e.g., errorDeafults typos)
+
+// Runs a block using common block/controller pattern
+function runBlock (config, done) {
+  // Set config.done if not provided
+  if (!config.done) {
+    config.done = done || (() => {
+    })
```

Comment:
The linter complained about the single line lambda. Not sure why and it reformatted itself to this.

`config.done` could be set directly so it would need to be `config.done = config.done || done || () => { }` which got a little long. I also felt the intent was more obvious with the `if`.